src/Client/OfficeInterop/OfficeInterop.fs

```diff
@@ -66,6 +66,27 @@ module OfficeInteropExtensions =
                 values  = U4.Case1 col
             )
 
+
+
+        static member adoptTableFormats (table:Table, context:RequestContext) =
```

Comment:
I had some function which did something similiar. This will be important for another one of the quickaccess buttons. Add a parameter which toggles hide or unhide.

if true -> hide all reference columns
if false -> unhide all columns

Reply:
fixed